src/expound/alpha.cljc

```diff
@@ -201,7 +201,9 @@
   ([size s label-str]
    (ansi/color
     (let [prefix (str label-str label-str " " s " ")
-          chars-left (- size (count prefix))]
+          chars-left (- #?(:clj  (long size)
```

Comment:
I favored `(long` over `^long` (which would also work) in face of:

![image](https://user-images.githubusercontent.com/1162994/71116989-82892c80-21d5-11ea-8852-313044afbfdf.png)

Reply:
Btw, the proposed code isn't either faster or slower (AFAICT): both imply a coercion. 

In my version, that coercion is explicit to humans, and in the old version `Clojure.RT` performs that coercion, under the hood